src/pages/docs/concepts-workers.mdx

```diff
@@ -0,0 +1,183 @@
+financial transactions, and other use cases that are not well-suited to
+traditional serverless platforms.
+
+## Comparing Functions to Workers
+
+|               | Worker | Function | Explanation                                                                                                           |
+| :------------ | :----: | :------: | :-------------------------------------------------------------------------------------------------------------------- |
+| Low-Latency   |   ✅   |    ✅    | Functions in serverless environments are designed to execute quickly, making them suitable for low-latency use cases. |
+| Scalable      |   ✅   |    ✅    | Functions in serverless environments scale automatically, making them suitable for high-throughput use cases.         |
+| Stateful      |   ✅   |    ❌    | Workers are inherently stateful, which means they maintain state for their lifetime, and across repeated invocations. |
+| Long-Running  |   ✅   |    ❌    | Workers run indefinitely, without loss of state or progress, making them uniquely suitable for long-running tasks.    |
+| Transactional |   ✅   |    ❌    | Workers are executed with strong transactional guarantees, transparently surviving faults, restarts, and updates.     |
+| Persistent    |   ✅   |    ❌    | All worker state, including in-memory state, is persistent and survives failures, restarts, and updates without loss. |
+
+## Emulating Classic Serverless
+
+Because Golem supports workers that run forever, each with their own unique
+identity, it may not be clear how to use Golem as a classic serverless platform.
+
+To emulate classic serverless, you really only need to do two things:
+
+1. **One-Export Component**. While WASM components can have any number of
+   exports, when emulating classic serverless, you should only have one export
+   per component. This export represents the event or request handler that you
+   would typically have in a classic serverless function.
+2. **One Worker Per Request**. When dispatching events or requests to your
+   component, you should create a new worker for each event or request. The
+   worker will handle the event or request, and never be used again.
+
+To ensure one worker per request or event, you can create a random worker name
+for each request or event, and use that to dispatch the event or request to the
+worker with the specified identity.
+
+Although the worker will remain available indefinitely, it will only be executed
+a single time, when used to handle the event or request. This allows you to
+emulate classic serverless behavior, while still benefiting from the durability
+and reliability of Golem.
+
+## Operations
+
+Workers support the following operations:
+
+- **Creation**. Workers benefit from automatic creation, which occurs when a
+  worker is invoked for the first time. Therefore, it is not necessary to
+  create workers explicitly.
+- **Cancellation**. Workers can be cancelled at any time, which causes the
```

Comment:
We don't have such a cancellation operation. What we have is **interruption**, and it's not corrupting the state. It behaves the same as a retryable error, except the executor is not scheduling a retry for it - so the interrupted worker remains idle until it gets an explicit **resume** call, or a new incoming invocation brings it in memory. In both cases it recovers the worker and continues running the previously interrupted invocation.

Reply:
Fixed.